frontend/packages/console-shared/src/components/pod/PodStatus.tsx

```diff
@@ -71,11 +80,45 @@ class PodStatus extends React.Component<PodStatusProps, PodStatusState> {
     return { vData, updateOnEnd: dataPoints === 1 && prevDataPoints > 1 };
   }
 
+  componentWillUnmount() {
+    this.unmounted = true;
+  }
+
   doUpdate = () => {
     // Animations complete, update to remove empty slices
     this.forceUpdate();
   };
 
+  onSegmentEnter = (index: number) => {
+    if (this.leaveTimer) {
+      clearTimeout(this.leaveTimer);
+      this.leaveTimer = null;
+    }
+
+    if (!this.enterTimer) {
+      this.enterTimer = setTimeout(() => {
+        if (!this.unmounted) {
+          this.setState(({ vData }) => ({ tipActive: true, tipData: vData[index] }));
```

Comment:
Storing the tipData like this, won't it cause the tooltip data to become stale if the user hovers the chart and the data starts changing? The current implementation keeps updating the data.
DIdn't test to confirm though.

Reply:
@christianvogt It could for sure. I can store the index or the x value. Either has the potential of going away or not being at the cursor position after the data updates though. I'm thinking the x value is probably best and it may just show a 0 value or not be what the user is hovering over any more.